agenta-cli/agenta/sdk/agenta_decorator.py

```diff
@@ -374,3 +376,10 @@ def find_in_schema(schema: dict, param_name: str, xparam: str):
         if isinstance(param_val, TextParam):
             subschema = find_in_schema(schema_to_override, param_name, "text")
             subschema["default"] = param_val
+        if (
+            isinstance(param_val, inspect.Parameter)
+            and param_val.annotation is MessagesParam
+        ):
+            subschema = find_in_schema(schema_to_override, param_name, "messages")
+            del subschema["items"]
+            subschema["default"] = param_val.default
```

Comment:
I don't think we need defaults for inputs

Reply:
We don't. Thank you very much!

Reply:
Edit: We actually do. I'll send you what I discovered on Slack!